apps/api/src/domain/models/__tests__/utilities/dummyIsbn.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * This is primitive (passed by value, not reference),
+ * so there is no need to use a 'builder' pattern.
+ */
+export const dummyIsbn = `978-3-16-148410-0`;
```

Comment:
It occurs to me that we might ideally want to just grab these things from the `FuzzGenerator`.

Reply:
That would be good